src/valkey-cli.c

```diff
@@ -3545,6 +3545,12 @@ static int evalMode(int argc, char **argv) {
         /* Call it */
         int eval_ldb = config.eval_ldb; /* Save it, may be reverted. */
         retval = issueCommand(argc + 3 - got_comma, argv2);
+
+        for (j = 0; j < argc + 3 - got_comma; j++) {
+            sdsfree(argv[j]);
```

Comment:
```suggestion
            sdsfree(argv2[j]);
```

Reply:
@flowerysong You're right. Thanks. applied